modules/microshift-config-cli-manifests.adoc

```diff
@@ -0,0 +1,154 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_content-type: CONCEPT
+[id="microshift-config-cli-manifests_{context}"]
+= Using CLI tools and creating manifests
+
+Configure your {product-title} using the supported command line arguments and environment variables.
+
+== Supported command-line arguments and environment variables
+
+[cols="4",options="header"]
+|===
+|Field Name
+|CLI Argument
+|Environment Variable
+|Description
+
+|clusterCIDR
+|--cluster-cidr
+|MICROSHIFT_CLUSTER_CLUSTERCIDR
+|A block of IP addresses from which Pod IP addresses are allocated
+
+|serviceCIDR
+|--service-cidr
+|MICROSHIFT_CLUSTER_SERVICECIDR
+|A block of virtual IP addresses for Kubernetes services
+
+|serviceNodePortRange
+|--service-node-port-range
+|MICROSHIFT_CLUSTER_SERVICENODEPORTRANGE
+|The port range allowed for Kubernetes services of type NodePort
+
+|dns
+|--cluster-dns
+|MICROSHIFT_CLUSTER_DNS
+|The Kubernetes service IP address where pods query for name resolution
+
+|domain
+|--cluster-domain
+|MICROSHIFT_CLUSTER_DOMAIN
+|Base DNS domain used to construct fully qualified pod and service domain names
+
+|url
+|--url
+|MICROSHIFT_CLUSTER_URL
+|URL of the API server for the cluster
+
+|nodeIP
+|--node-ip
+|MICROSHIFT_NODEIP
+|The IP address of the node, defaults to IP of the default route
+
+|nodeName
+|--node-name
+|MICROSHIFT_NODENAME
+|The name of the node, defaults to hostname
+
+|logVLevel
+|--v
+|MICROSHIFT_LOGVLEVEL
+|Log verbosity (0-5)
+|===
+
+== Auto-applying Manifests
```

Comment:
This might be worth turning into its own module. It's not really about configuring MicroShift itself, and is more about deploying a workload on MicroShift.

Reply:
I see your point, hhhmm, should we create a "Manage" or "Deploying"/"Deploying Workloads" book and start it with this module? We're not really managing, per se? We could fold in the procedures to observe as well.

Reply:
I don't know if we would have more to add to a "Deploying Workloads" section. Maybe we can reuse some of what goes into https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html/building_applications/deployments though?